src/packageManager.ts

```diff
@@ -32,7 +32,9 @@ export class PackageManager extends EventEmitter {
         this.storage = storage;
         this.packages = {};
         this.version = version;
-        this.pckdir_watch = chokidar.watch(storage);
+        this.pckdir_watch = chokidar.watch(storage, {
+            depth: 1,
```

Comment:
Hi, thanks for the efford. I have just tested your solution and it doesn't work for file /localdata/settings.json (used by UI for micro applications). I think the correct solution should be:
```
this.pckdir_watch = chokidar.watch(storage, {
    ignored: `${storage}/**/node_modules/**`,
});
```

Reply:
Yes, that is an alternative that works too. It will still create watchers for more files than needed, but is better than the current situation.

It seems possible to add more than one path to the `watch()` function. Do you think the following works too? I have not tested it out yet, but I would prefer to watch only the directories/files that are required, and not include any micro-app files (like `html/**`).

```js
this.pckdir_watch = chokidar.watch((storage, `${storage}/**/localdata/**`), {
    depth: 1,
});
```

Reply:
It should be defined as array:
```
 [storage, `${storage}/**/localdata/**`]` 
```
but it doesn't work for me either. Maybe there is a problem in the library when one path is subset of the other. 

I don't think too many watchers are huge problem if node_modules are excluded but another and maybe easiest solution is to create two watchers with depth=1. One for the `packages` directory and the other one for the `${storage}/**/localdata/settings.json` files. It requires slightly bigger change of the code.

Reply:
It is solved in commit:  [2f7c30ef358f6ea5567359f29093f1daecf45a00](https://github.com/CamStreamer/CamScripter_Raspberry_Pi/commit/2f7c30ef358f6ea5567359f29093f1daecf45a00)